src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const Home = () => {
+  return <section>Home page</section>;
```

Comment:
This is the game homepage?

Reply:
Yes, that would be the page after login and team existence.